data/data/openstack/main.tf

```diff
@@ -28,7 +32,7 @@ module "service" {
 
   swift_container   = "${openstack_objectstorage_container_v1.container.name}"
   cluster_id        = "${var.cluster_id}"
-  cluster_domain    = "${var.cluster_domain}"
```

Comment:
`cluster_name` is no longer available since https://github.com/openshift/installer/commit/5b06cb265cf3004e809d0b0319530e8e179c6cdb from https://github.com/openshift/installer/pull/1280